common-auth/src/main/java/com/hortonworks/registries/auth/server/JWTAuthenticationHandler.java

```diff
@@ -147,6 +149,14 @@ public void init(Properties config) throws ServletException {
         if (customCookieName != null) {
             cookieName = customCookieName;
         }
+
+        nonBrowserUserAgents = config.getProperty(
```

Comment:
Can we avoid duplicating code between handler classes? maybe we can create  getNonBrowserUserAgents() util method?